packages/devextreme/js/__internal/ui/html_editor/ui/aiDialog.ts

```diff
@@ -21,26 +21,46 @@ const AI_DIALOG_CONTROLS_CLASS = 'dx-aidialog-controls';
 const AI_DIALOG_CONTENT_CLASS = 'dx-aidialog-content';
 const AI_DIALOG_TITLE_CLASS = 'dx-aidialog-title';
 const AI_DIALOG_TITLE_TEXT_CLASS = 'dx-aidialog-title-text';
+const ICON_CLASS = 'dx-icon';
+const ICON_SPARKLE_CLASS = 'dx-icon-sparkle';
 
 const POPUP_MIN_WIDTH = 288;
 const POPUP_MAX_WIDTH = 460;
 const REPLACE_DROPDOWN_WIDTH = 150;
+const TEXT_AREA_MIN_HEIGHT = 64;
```

Comment:
I'm not sure it's needed. It looks better for me if we don't specify minHeight. 
However, you can merge it this way and store point to next task if you want